src/components/WebexMediaAccess/WebexMediaAccess.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  Icon,
+} from '@momentum-ui/react';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+
+/**
+ * Webex Media Access component displays a prompt explaining necessary access to media.
+ *
+ * @param {object} props  Data passed to the component
+ * @param {Function} joinWithoutCamera This function is triggered by clicking the "Join without camera" button
```

Comment:
- double space between arg name and description
- use `props.joinWithoutCamera` as the arg name